wireless_modules/middle_module/main.py

```diff
@@ -28,6 +28,13 @@
 sensors = [my_mpu, my_dht, my_mq135]
 middle_module.add_sensors(sensors)
 
-# Enters an infinite loop
-print("starting loop")
-middle_module.run()
+
+async def main():
+    print("Starting asyncio loop")
+    asyncio.create_task(middle_module.run())
+
+    while True:
+        await asyncio.sleep(1)
```

Comment:
Might help explain what's happening here with in-line comments, since to someone who's new to `asyncio` (me for example) this code looks redundant